cmd/goal/network.go

```diff
@@ -47,6 +48,7 @@ func init() {
 	networkCreateCmd.Flags().BoolVarP(&noImportKeys, "noimportkeys", "K", false, "Do not import root keys when creating the network (by default will import)")
 	networkCreateCmd.Flags().BoolVar(&noClean, "noclean", false, "Prevents auto-cleanup on error - for diagnosing problems")
 	networkCreateCmd.Flags().BoolVar(&devModeOverride, "devMode", false, "Forces the configuration to enable DevMode, returns an error if the template is not compatible with DevMode.")
+	networkCreateCmd.Flags().BoolVar(&noRewardOverride, "noReward", false, "Forces the configuration to disable Rewards")
```

Comment:
@barnjamin A few thoughts here:

* As an alternative approach, I'm wondering if you considered extending the network template to support configuring the _RewardsPool_ initial balance?
  * I considered the idea while inspecting the PR.  From what I see, it feels like exposing the initial balance configuration is a lower friction change.  And I can imagine template generators operating at a higher level of abstraction providing a _don't fund rewards_ convenience configuration.
  * Open for discussion:  For convenience, I can imagine 0 being coerced to the current consensus protocol's min balance value. 
  * A possible complication:  Depending on how the change is reflected in JSON, it _might_ not be backwards compatible.  I'm unfamiliar with backwards compatibility concerns here.  We can discuss live should there be interest in pursuing the idea.
* Optional - If we keep the PR's approach, then I propose renaming from `noReward` to something like `noInitialRewardPoolFunding` (open to ideas).  I think language like _no rewards_ and _disable rewards_ can be misleading because the account _can be_ funded after genesis.

Reply:
> As an alternative approach, I'm wondering if you considered extending the network template to support configuring the RewardsPool initial balance?

This is a fair point but I think might complicate the setup process more than it helps. If the dev wants to have rewards they can just send rewards to the rewards pool account though it will need to be documented.

> Optional - If we keep the PR's approach, then I propose renaming from noReward to something like noInitialRewardPoolFunding (open to ideas). I think language like no rewards and disable rewards can be misleading because the account can be funded after genesis.

This is closer to the behavior I think we should use. For the name of the flag, this seems like a fine choice and unless you or anyone reading this has a better suggestion I'll make it so

Reply:
I ended up using `noInitialRewards*` as the name since the `funding` bit complicates the concept I think, though I did try to describe what is happening in the cli flag

Reply:
@barnjamin Let me know if the note is convincing?

I won't stand in the PR's way, but I'm not convinced the PR presents the best tradeoff.  If my note isn't convincing, then I'll follow the PR assuming a boolean flag will be added.

Primary concern:  
* Over time, it's plausible to see a proliferation of boolean flags that create branching without providing direct access to underlying configuration.  It _might_ feel manageable today, but will grow more problematic over time.  
* I don't have much prior experience with genesis template formulation, so it's possible my viewpoint is in left field.  My concern arises from past experiences managing analogous configs.

From my perspective:
* For genesis template, instead of exposing a boolean flag, I think it ought to support overriding _RewardsPool_ balance.
* I'd probably _not_ expose a `goal network create` flag.  Though, a possible compromise is to keep the boolean CLI flag and modify its behavior to override the initial balance.
  * The compromise retains direct access to configuration at the lowest level (genesis template) while providing conveniences at a higher level (`goal network create`).

Reply:
Took out the create flag in favor of a genesis config template field to specify rewards pool balance

Reply:
@barnjamin Resolving - Thanks for taking in the feedback.